CHANGELOG.md

```diff
@@ -21,6 +21,7 @@ listed in the changelog.
 - Add more context to Bitbucket client errors ([#515](https://github.com/opendevstack/ods-pipeline/issues/515))
 - Update skopeo to 1.6, buildah to 1.24 and git to 2.31 ([#519](https://github.com/opendevstack/ods-pipeline/issues/519))
 - Update Go to 1.17 ([#528](https://github.com/opendevstack/ods-pipeline/issues/528))
+- Implement global caching for Gradle build task ([#490](https://github.com/opendevstack/ods-pipeline/issues/490))
```

Comment:
I think we should add a separate entry in the `Fixed` section that describes the fix to the HTTP proxy handling and links to this pull request.